apps/autoscheduler/cost_model_schedule.h

```diff
@@ -0,0 +1,708 @@
+    Func repeat_edge_1 = p.get_func(5);
+    Func sum_1_1_d_def__ = p.get_func(4);
+    Func constant_exterior = p.get_func(3);
+    Func repeat_edge = p.get_func(2);
+    Func loss_output_0_d_def__ = p.get_func(1);
+    Func adjoint = p.get_func(0);
+    Var c(constant_exterior_10.get_schedule().dims()[0].var), ci("ci"), cii("cii"), n(f2.get_schedule().dims()[0].var), ni("ni"), r1001_y(head2_relu_0_d_def__.update(0).get_schedule().dims()[1].var), r1025_z(filter1_im_0_d_def__.update(0).get_schedule().dims()[2].var), r1110_z(head2_filter_im_0_d_def__.update(0).get_schedule().dims()[2].var), r1198_y(head1_conv_1_d_def__.update(0).get_schedule().dims()[1].var), r1222_y(filter1_im_0_d_def__.update(1).get_schedule().dims()[1].var), r1303_x(squashed_head1_filter_broadcast_0_d_def__.update(0).get_schedule().dims()[0].var), r1303_y(squashed_head1_filter_broadcast_0_d_def__.update(0).get_schedule().dims()[1].var), r189_x(conv1_stage2_1_d_def__.update(0).get_schedule().dims()[0].var), r189_y(conv1_stage2_1_d_def__.update(0).get_schedule().dims()[1].var), r217_x(f0_0_d_def__.update(0).get_schedule().dims()[0].var), r96_x(f2_0_d_def__.update(0).get_schedule().dims()[0].var), s(constant_exterior_18.get_schedule().dims()[1].var), v2947(updated_head1_filter.get_schedule().dims()[0].var), v2947i("v2947i"), v2948(updated_head1_filter.get_schedule().dims()[1].var), v2949(updated_head1_filter.get_schedule().dims()[2].var), v2950(updated_head1_filter.get_schedule().dims()[3].var), v2971(updated_head1_bias.get_schedule().dims()[0].var), v2971i("v2971i"), v2972(updated_head1_bias.get_schedule().dims()[1].var), v2983(updated_head2_filter.get_schedule().dims()[0].var), v2983i("v2983i"), v2983ii("v2983ii"), v2984(updated_head2_filter.get_schedule().dims()[1].var), v2985(updated_head2_filter.get_schedule().dims()[2].var), v3001(updated_head2_bias.get_schedule().dims()[0].var), v3001i("v3001i"), v3002(updated_head2_bias.get_schedule().dims()[1].var), v3013(updated_filter1.get_schedule().dims()[0].var), v3013i("v3013i"), v3013ii("v3013ii"), v3014(updated_filter1.get_schedule().dims()[1].var), v3015(updated_filter1.get_schedule().dims()[2].var), v3031(updated_bias1.get_schedule().dims()[0].var), v3031i("v3031i"), v3032(updated_bias1.get_schedule().dims()[1].var), v360(repeat_edge_23.get_schedule().dims()[0].var), v360i("v360i"), v364(constant_exterior_22.get_schedule().dims()[0].var), v364i("v364i"), v365(constant_exterior_22.get_schedule().dims()[1].var), v369(head1_bias_im_0_d_def__.get_schedule().dims()[0].var), v369i("v369i"), v383(constant_exterior_19.get_schedule().dims()[0].var), v383i("v383i"), v384(constant_exterior_19.get_schedule().dims()[1].var), v385(constant_exterior_19.get_schedule().dims()[2].var), v420(repeat_edge_14.get_schedule().dims()[0].var), v420i("v420i"), v424(constant_exterior_13.get_schedule().dims()[0].var), v424i("v424i"), v425(constant_exterior_13.get_schedule().dims()[1].var), w(constant_exterior_10.get_schedule().dims()[1].var), wi("wi");
```

Comment:
I would gently vote that the generating code should emit one Var/RVar per line rather than this ginormous line

Reply:
done